packages/reakit-test-utils/src/verify.ts

```diff
@@ -0,0 +1,8 @@
+import { configureAxe } from "jest-axe";
+
+export const verify = configureAxe({
```

Comment:
Naming it `axe` seems to be the convention. I'd suggest to rename it so (and also rename the file). But let me know if there's any other reason to name it differently.

Reply:
Cools, happy to change it to `axe` 👌🏻